internal/service/dns_config/api_forward_zone_resource_test.go

```diff
@@ -0,0 +1,568 @@
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckForwardZoneExists(context.Background(), resourceName, &v),
+					resource.TestCheckResourceAttr(resourceName, "external_forwarders.0.address", "192.168.10.10"),
+					resource.TestCheckResourceAttr(resourceName, "external_forwarders.0.fqdn", "tf-infoblox-test.com."),
+				),
+			},
+			// Update and Read
+			{
+				Config: testAccForwardZoneExternalForwarders(fqdn, "192.168.11.11", "tf-infoblox.com."),
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckForwardZoneExists(context.Background(), resourceName, &v),
+					resource.TestCheckResourceAttr(resourceName, "external_forwarders.0.address", "192.168.11.11"),
+					resource.TestCheckResourceAttr(resourceName, "external_forwarders.0.fqdn", "tf-infoblox.com."),
+				),
+			},
+			// Delete testing automatically occurs in TestCase
+		},
+	})
+}
+
+func TestAccForwardZoneResource_ForwardOnly(t *testing.T) {
+	var resourceName = "bloxone_dns_forward_zone.test_forward_only"
+	var fqdn = acctest.RandomNameWithPrefix("fw-zone") + ".com."
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:                 func() { acctest.PreCheck(t) },
+		ProtoV6ProviderFactories: acctest.ProtoV6ProviderFactories,
+		Steps: []resource.TestStep{
+			// Create and Read
+			{
+				Config: testAccForwardZoneForwardOnly(fqdn, "false"),
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttr(resourceName, "forward_only", "false"),
+				),
+			},
+			// Update and Read
+			{
+				Config: testAccForwardZoneForwardOnly(fqdn, "true"),
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttr(resourceName, "forward_only", "true"),
+				),
+			},
+			// Delete testing automatically occurs in TestCase
+		},
+	})
+}
+
+func TestAccForwardZoneResource_Hosts(t *testing.T) {
+	var resourceName = "bloxone_dns_forward_zone.test_hosts"
+	var fqdn = acctest.RandomNameWithPrefix("fw-zone") + ".com."
+	var v dns_config.ConfigForwardZone
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:                 func() { acctest.PreCheck(t) },
+		ProtoV6ProviderFactories: acctest.ProtoV6ProviderFactories,
+		Steps: []resource.TestStep{
+			// Create and Read
+			{
+				Config: testAccForwardZoneHosts(fqdn),
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckForwardZoneExists(context.Background(), resourceName, &v),
+					resource.TestCheckResourceAttr(resourceName, "hosts.#", "1"),
+					resource.TestCheckResourceAttrPair(resourceName, "hosts.0", "data.bloxone_dns_hosts.test_host_01", "results.0.id"),
+				),
+			},
+			// Update and Read
+			{
+				Config: testAccForwardZoneHosts(fqdn),
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckForwardZoneExists(context.Background(), resourceName, &v),
+					resource.TestCheckResourceAttr(resourceName, "hosts.#", "1"),
+					resource.TestCheckResourceAttrPair(resourceName, "hosts.0", "data.bloxone_dns_hosts.test_host_01", "results.0.id"),
+				),
+			},
```

Comment:
I don't see any updates here. If there is nothing to be updated than this case can be removed or you can put a to do item for updating the host when you have the second host available.

Reply:
fixed